src/pages/Settings/Settings.js

```diff
@@ -1,146 +1,182 @@
-})
-
-export function Settings ({
-  settings,
-  updateSettings,
-  history: { push },
-  toggleSshAccess
-}) {
-  const { register, handleSubmit, errors } = useForm({
-    defaultValues: settings,
-    validationSchema: SettingsValidationSchema
+// Data - Mutation hook
+function useUpdateVersion () {
+  const [hposUpdateVersion] = useMutation(HposUpdateVersionMutation)
+  return (availableVersion) => hposUpdateVersion({
+    variables: { availableVersion }
   })
+}
 
-  const [sshAccessVal, setSshAccess] = useState(false)
-  const handleToggleSshAccess = (e) => {
-    e.preventDefault()
-    setSshAccess(e.target.checked)
-    toggleSshAccess()
-  }
+export function Settings ({ history: { push } }) {
+  const { data: { hposSettings: settings = [] } = {} } = useQuery(HposSettingsQuery)
+  const { data: { hposStatus: status = [] } = {} } = useQuery(HposStatusQuery)
+
+  const updateVersion = useUpdateVersion()
+  const [softwareUpdateVersion, setSoftwareUpdateVersion] = useState()
+  const showSoftwareUpdateModal = availableVersion => setSoftwareUpdateVersion(availableVersion)
 
-  const onSubmit = settings => {
-    updateSettings(settings)
+  let updateAvailable = false
+  let availableVersion, currentVersion
+  if (!isEmpty(status) && !isEmpty(status.versionInfo)) {
+    availableVersion = status.versionInfo.availableVersion
+    currentVersion = status.versionInfo.currentVersion
   }
+  if (!isEmpty(availableVersion) && !isEmpty(currentVersion) && (availableVersion !== currentVersion)) updateAvailable = true
 
   return <PrimaryLayout headerProps={{ title: 'HoloPort Settings' }}>
-    <strong style={{ marginTop: '20px' }}>Name</strong>
-    <p>
-      {settings.deviceName}
-    </p>
-
-    <strong>URL</strong>
-    <p>
-      {settings.hostUrl}
-    </p>
-
-    <strong>Network ID</strong>
-    <p>
-      {settings.networkId}
-    </p>
-
-    <strong>Access Port Numbers</strong>
-
-    <form styleName='settings-form' onSubmit={handleSubmit(onSubmit)}>
-      <SettingsFormInput
-        label='Device Admin'
-        name='deviceAdminPort'
-        register={register}
-        errors={errors} />
-
-      <SettingsFormInput
-        label='HC Admin'
-        name='hcAdminPort'
-        register={register}
-        errors={errors} />
-
-      <SettingsFormInput
-        label='HC Network'
-        name='hcNetworkPort'
-        register={register}
-        errors={errors} />
-
-      <SettingsFormInput
-        label='Hosting'
-        name='hostingPort'
-        register={register}
-        errors={errors} />
-
-      <Button variant='primary' wide name='update-settings' value='Submit'>Save Changes</Button>
-    </form>
+    <header styleName='jumbotron-header'>
+      {!isEmpty(settings) && <>
+        <HashAvatar seed={settings.hostPubKey} styleName='avatar-image' />
+        <h2> {settings.hostName ? `${settings.hostName}'s` : 'Your'} HoloPort </h2>
+      </> }
+      {/* TODO: Find out what the below number should link to... */}
```

Comment:
is this captured in pivotal?

Reply:
Note is now both in the wireframes and Pivotal.